src/components/Header/index.tsx

```diff
@@ -402,59 +409,48 @@ export default function Header() {
               <Trans>Swap</Trans>
               <DropdownIcon />
             </Flex>
-
-            <Dropdown id={TutorialIds.BRIDGE_LINKS}>
```

Comment:
maybe conflict when rebase/merge
help me update to `<Dropdown id=  id={TutorialIds.BRIDGE_LINKS}>` and remove div `<div id={TutorialIds.BRIDGE_LINKS}>`

Reply:
`id= id=` mistake?